### ISSUE TITLE
Register finalized Python releases with GitHub Releases

### ISSUE THREAD

**Issue by mcandre** [python/cpython#123444]
# Feature or enhancement

### Proposal:

As an engineer, I want Python to promote the finalized (not alpha or beta) version tags to be promoted to GitHub Releases, so that I can subscribe to RSS/Atom feeds to keep up to date with the latest Python security patches and features.

### Has this already been discussed elsewhere?

This is a minor feature, which does not need previous discussion elsewhere

**Comment by hugovk** [comment]
I can see the value in doing this.

However, I suggest also including pre-releases, because we really do want more people to know about them, test them and provide early feedback. 

There's a checkbox specifically for it:

> * [ ] **Set as a pre-release**
>       This release will be labeled as non-production ready

https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

**Comment by mcandre** [comment]
Sure, as long as nonstable versions are tracked in a separate RSS/Atom feed. I don't want the noise of additional notifications since I have a strong preference for only interacting with stable, ideally LTS releases of my tech stack components.

**Comment by ned-deily** [comment]
There is an open issue about providing an [RSS/Atom feed for releases via the python.org website](https://github.com/python/pythondotorg/issues/1612).

**Comment by mcandre** [comment]
Not quite. There is already a feed on GitHub for tags, where the discussion is headed. But that mushes together unstable versions with stable versions.

Simply promoting stable tags to GitHub Releases would solve the problem.

Linking to the release feed on python.org is a bonus.
